File: src/main/java/br/com/blz/testjava/domain/Warehouse.java
package br.com.blz.testjava.domain;

import br.com.blz.testjava.api.v1.model.WarehouseType;
import lombok.Builder;
import lombok.Data;

@Data @Builder
public class Warehouse {
	private String locality;
	private int quantity;
	private WarehouseType type;
}

File: src/main/kotlin/br/com/blz/testjava/domain/Warehouse.kt
package br.com.blz.testjava.domain

import br.com.blz.testjava.domain.enums.Locality
import br.com.blz.testjava.domain.enums.WarehouseType

class Warehouse(
  val anLocality: Locality,
  val quantity: Int,
  val type: WarehouseType
)
